docs/manual/docs/cmd/graph/schemaextension/schemaextension-set.md

```diff
@@ -0,0 +1,60 @@
+# graph schemaextension add
+
+Updates a Microsoft Graph schema extension
+
+## Usage
+
+```sh
+graph schemaextension set [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-i, --id <id>`|The unique identifier for the schema extension definition
+`-d, --description [description]`|Description of the schema extension
+`-s, --status [status]`|The lifecycle state of the schema extension. Accepted values are 'Available' or 'Depecrated'
+`-t, --targetTypes <targetTypes>`|Comma-separated list of Microsoft Graph resource types the schema extension targets
+`-p, --properties`|The collection of property names and types that make up the schema extension definition formatted as a JSON string
+`-o, --output [output]`|Output type. `json|text`. Default `text`
+`--verbose`|Runs command with verbose logging
+`--debug`|Runs command with debug logging
+
+## Remarks
+
+The lifecycle state of the schema extension. The initial state upon creation is `InDevelopment`.
+Possible states transitions are from `InDevelopment` to `Available` and `Available` to `Deprecated`.
+
+The target types are the set of Microsoft Graph resource types (that support schema extensions) that this schema extension definition can be applied to. This option is specified as a comma-separated list
```

Comment:
Unnecessary space in front

Reply:
Not sure which space you meant ?

Reply:
Just right in front of `T` at the beginning of line 30

Reply:
Okay, I fixed it already, let me know if there is anything else :)